src/Month.java

```diff
@@ -0,0 +1,27 @@
+public class Month {
+    public static void main(String[] args) {
```

Comment:
Please implement it in a separate method (as you wrote in the Max.class). In the main method you can use the method. Because now your code works for the 4th month only. So, create a new method with a month as a parameter and then use the method in the main method:
yourMethod(4); // or "April" if you want to use String